build.rs

```diff
@@ -117,7 +117,7 @@ mod c {
 
         cfg.warnings(false);
 
-        if target_env == "msvc" {
+        if target_env == "msvc" && !target_arch_arm {
```

Comment:
Hm is this right? This means that if we compile anything it'll use gcc-style flags, right?

Reply:
You're right, this change is incorrect.

Reply:
Undid the unnecessary changes. Let me know if you'd like me to squash.